src/middleware/missingRedirectErrorHandler.test.js

```diff
@@ -0,0 +1,40 @@
+const {
+  missingRedirectErrorHandler,
+} = require("./missingRedirectErrorHandler");
+
+describe("Missing Redirect Error Handler Middleware", () => {
+  let err, req, res, next;
+
+  beforeEach(() => {
+    err = sinon.stub();
+    req = sinon.fake();
+    res = {
+      status: sinon.fake(),
```

Comment:
You can add these functions into the globals as defined in `mocha-helpers` https://github.com/govuk-one-login/ipv-cri-address-front/blob/main/test/utils/mocha-helpers.js#L53 - then you can just run `setupDefaultMocks()`

(this utility should be extracted out for easier reuse, but that's also a refactoring for another day)